src/app/components/gv-page-asyncapi/gv-page-asyncapi.component.html

```diff
@@ -0,0 +1,35 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+
+<div class="gv-page-container asyncapi">
+  <div ng-if="page.content">
+    <asyncapi-component
+      schema="{{page.content}}"
+      cssImportPath="assets/styles/asyncapi-component.css">
+    </asyncapi-component>
+  </div>
```

Comment:
I think `div` can be merged with `asyncapi-component`:
```html
    <asyncapi-component
      ng-if="page.content"
      schema="{{page.content}}"
      cssImportPath="assets/styles/asyncapi-component.css">
    </asyncapi-component>

```

Reply:
done